test/parallel/test-http-client-response-domain.js

```diff
@@ -45,7 +45,7 @@ server.listen(common.PIPE, function() {
 function test() {
 
   d.on('error', common.mustCall(function(err) {
```

Comment:
@sheminusminus - just a suggestion: if you like, you could also replace this call back (`function(err)...`) into an arrow function (`(err) => .. `) if you like, but feel free to ignore

Reply:
@gireeshpunathil made the update!